package.json

```
@@ -3,8 +3,10 @@
   "version": "3.0.0",
   "description": "Request raw body",
   "main": "plugin.js",
+  "types": "plugin.d.ts",
   "scripts": {
-    "test": "standard && tap test/**.test.js"
+    "test": "standard && tap test/**.test.js",
+    "typescript": "tsd"
```

Added in the test run, so the CI will execute it

```suggestion
    "test": "standard && tap test/**.test.js && tsd"
```

---

Done 😄